### PR TITLE
Add Kubernetes deployment for Edge

### PR DESCRIPTION
**Pull request by edisonxiang**
**What type of PR is this?**
 /kind test

**What this PR does / why we need it**:
* fix some notes in dockercompose
* add kubernetes deployment for Edge

### CODE REVIEW

File: build/edge/README.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ container and MQTT Broker, so make sure that docker engine listening on
   ```shell
   ./build/edge/run_daemon.sh set \
   		    cloudhub=0.0.0.0:10000 \
```
Comment (m1093782566) [inline build/edge/README.md]: Please remove some unnecessary whitespaces in this line?

Comment (edisonxiang) [reply build/edge/README.md]: I guess we need it like the following line ` Optional: x86_64 \| aarch` :)

File: build/edge/README_zh.md [hunk 2]
```diff
@@ -26,13 +26,13 @@
   ```shell
   ./build/edge/run_daemon.sh set \
   		    cloudhub=0.0.0.0:10000 \
```
Comment (m1093782566) [inline build/edge/README_zh.md]: Same here.

Comment (edisonxiang) [reply build/edge/README_zh.md]: Got it. Done

File: build/edge/README.md (hunk 1 above)
Comment (edisonxiang) [reply build/edge/README.md]: Done

**Comment by m1093782566** [comment]
Please remove WIP when you think it's ready to go.

**Comment by edisonxiang** [comment]
Sure. Thanks @m1093782566

**Comment by m1093782566** [comment]
@pavan187 

Seems there are some flaky e2e test cases. Please check:

```
Summarizing 3 Failures:
[Fail] Application deployment test in E2E scenario Test application deployment and delete deployment using deployment spec [AfterEach] E2E_APP_DEPLOYMENT_1: Create deployment and check the pods are coming up correctly 
/home/travis/gopath/src/github.com/kubeedge/kubeedge/tests/e2e/utils/pod.go:138
[Fail] Application deployment test in E2E scenario Test application deployment and delete deployment using deployment spec [AfterEach] E2E_APP_DEPLOYMENT_2: Create deployment with replicas and check the pods are coming up correctly 
/home/travis/gopath/src/github.com/kubeedge/kubeedge/tests/e2e/utils/pod.go:138
[Fail] Application deployment test in E2E scenario Test application deployment and delete deployment using deployment spec [It] E2E_APP_DEPLOYMENT_3: Create deployment and check deployment ctrler re-creating pods when user deletes the pods manually 
/home/travis/gopath/src/github.com/kubeedge/kubeedge/tests/e2e/deployment/deployment_test.go:133
```

**Comment by edisonxiang** [comment]
@m1093782566 @pavan187 
PTAL. Thanks very much

File: build/edge/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+##  Deploy the edge part into a k8s cluster
```
Comment (m1093782566) [inline build/edge/kubernetes/README.md]: Can we create a new file and simply link the file in the README?

File: build/edge/kubernetes/03-configmap-edgenodecerts.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+kind: ConfigMap
+metadata:
+  name: edgenodecerts
+  namespace: kubeedge
+  labels:
+    k8s-app: kubeedge
+    kubeedge: edgenode
+data:
+  edge.crt: |
+    -----BEGIN CERTIFICATE-----
```
Comment (m1093782566) [inline build/edge/kubernetes/03-configmap-edgenodecerts.yaml]: Please generate the key and cert files at runtime.

File: build/edge/kubernetes/README_zh.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+## 部署 edge 端到 k8s 集群
```
Comment (m1093782566) [inline build/edge/kubernetes/README_zh.md]: Same as above.

File: edge/pkg/edged/dockertools/docker_manager.go [hunk 6]
```diff
@@ -73,7 +73,7 @@ const (
 )
 
 //DockerDefaultAddress is default address for docker
-const DockerDefaultAddress = "unix:///var/run/docker.sock"
+var DockerDefaultAddress = "unix:///var/run/docker.sock"
```
Comment (m1093782566) [inline edge/pkg/edged/dockertools/docker_manager.go]: Any reason to change const to var? For docker **default** address, I think it's OK to take it as const.

Comment (edisonxiang) [reply edge/pkg/edged/dockertools/docker_manager.go]: OK. I will define new variable for the DockerAddress

File: build/edge/kubernetes/README.md (hunk 3 above)
Comment (edisonxiang) [reply build/edge/kubernetes/README.md]: Yep

File: build/edge/kubernetes/README_zh.md (hunk 5 above)
Comment (edisonxiang) [reply build/edge/kubernetes/README_zh.md]: OK

**Comment by m1093782566** [comment]
Please rebase to the HEAD for running test jobs again.

**Comment by edisonxiang** [comment]
Done. @m1093782566 @pavan187 PTAL.

**Comment by m1093782566** [comment]
@edisonxiang Please squash the commits.

**Comment by edisonxiang** [comment]
Done. @m1093782566

**Pull request merged**